frontend/src/components/VariantListPage/VariantsTable.tsx

```diff
@@ -716,65 +828,37 @@ const VariantsTable: FC<VariantsTableProps> = ({
                     )}
                   </>
                 ) : (
-                  column.heading
+                  <span
+                    style={{
+                      position: "relative",
+                      top: "37px",
+                    }}
+                  >
+                    {column.heading}
+                  </span>
                 )}
               </Th>
             );
           })}
         </Tr>
       </Thead>
       <Tbody>
-        {sortedVariants.map((variant) => {
-          return (
-            <Tr key={variant.id}>
-              {includeCheckboxColumn && (
-                <Td>
-                  <Checkbox
-                    isChecked={selectedVariants.has(variant.id)}
-                    onChange={(e) => {
-                      if (e.target.checked) {
-                        onChangeSelectedVariants(
-                          new Set([...selectedVariants, variant.id])
-                        );
-                      } else {
-                        onChangeSelectedVariants(
-                          new Set(
-                            [...selectedVariants].filter(
-                              (variantId) => variantId !== variant.id
-                            )
-                          )
-                        );
-                      }
-                    }}
-                  >
-                    <VisuallyHidden>
-                      Include this variant in calculations
-                    </VisuallyHidden>
-                  </Checkbox>
-                </Td>
-              )}
-              {columns.map((column) => {
-                return (
-                  <Td
-                    key={column.key}
-                    as={column.key === "variant_id" ? "th" : undefined}
-                    scope={column.key === "variant_id" ? "row" : undefined}
-                    fontWeight="normal"
-                    isNumeric={column.isNumeric}
-                  >
-                    {column.render(
-                      variant,
-                      variantList,
-                      variantNotes,
-                      onEditVariantNote,
-                      userCanEdit
-                    )}
-                  </Td>
-                );
-              })}
-            </Tr>
-          );
-        })}
+        <FixedSizeList
+          height={10 * ROW_HEIGHT - 1}
```

Comment:
Two quick things here:

1. To make things more clear, we should take out the magic number `10` and put it into a more descriptive variable. This 10 drives how many items are displayed in the fixed size list (I realize I put this magic number in the `DashboardLists` component in the first place, and in looking at PR I had to think about what it does/represents, which is a good sign that it really should be an appropriately named variable)
2. It might be beneficial to display a few more rows, 15 or 20? I'd look at the local page on your default zoom, then zoom out a few times in the browser and see what feels appropriate. Since a typical variant list probably has at least 50, and possibly in the hundreds of variants, a few more in display may be useful, lowering the height of individual rows will get us part of the way there, then increasing the amount displayed at a time can get use closer/all the way there.